src/main/webapp/resources/js/pages/analysis/components/AnalysisError.jsx

```diff
@@ -1,12 +1,140 @@
-import React from "react";
-import { Typography } from "antd";
+/*
+ * This file renders the error details for the analysis
+ */
 
-const { Title } = Typography;
+/*
+ * The following import statements makes available all the elements
+ * required by the component encompassed within
+ */
+
+import React, { Suspense, useContext, useEffect, useState } from "react";
+import { AnalysisContext } from "../../../contexts/AnalysisContext";
+import { Alert, Spin, Tabs } from "antd";
+import { getJobErrors } from "../../../apis/analysis/analysis";
+import { getI18N } from "../../../utilities/i18n-utilties";
+import styled from "styled-components";
+
+const GalaxyJobInfoTab = React.lazy(() => import("./GalaxyJobInfoTab"));
+const GalaxyParametersTab = React.lazy(() => import("./GalaxyParametersTab"));
+const StandardErrorTab = React.lazy(() => import("./StandardErrorTab"));
+const StandardOutputTab = React.lazy(() => import("./StandardOutputTab"));
+
+const TabPane = Tabs.TabPane;
+
+export default function AnalysisError(props) {
+  const { analysisContext } = useContext(AnalysisContext);
+  const [jobErrors, setJobErrors] = useState(null);
+  const [currActiveKey, setCurrActiveKey] = useState(1);
+
+  const StyledTabs = styled(Tabs)`
```

Comment:
Can you, or did you, make this a proper component?

Reply:
I have in a branch that is based off this one.